ipam_test.go

```diff
@@ -828,7 +863,7 @@ func ExampleFree() {
 	// Output:
 	// 10.4.0.0/24
 	// 10.4.1.0/28
-	// 10.4.1.16/24
+	// 10.4.2.0/24
```

Comment:
@JosephSalisbury I believe this was accidentally wrong and there's nothing that specifically expects something like this?

Reply:
Yeah, that looks right to me. We obviously can't start a /24 at 10.4.1.16 :D